resources/js/store/modules/session.js

```diff
@@ -13,8 +13,30 @@ const getters = {
   isAuthenticated: state => {
     return !$.isEmptyObject(state.currentUser);
   },
+
   settings: (state) => (setting) => {
-    return $.isEmptyObject(state.settings) || !(setting in state.settings) ? null : state.settings[setting];
+    /** Get a nested property from an object without returning any errors.
+     * If the property or property chain doesn't exist, undefined is returned.
+     * Property names with spaces may use either dot or bracket "[]" notation.
+     * Note that bracketed property names without surrounding quotes will fail the lookup.
+     *      e.g. embedded variables are not supported.
+     * @param {object} obj The object to check
+     * @param {string} prop The property or property chain to get (e.g. obj.prop1.prop1a or obj['prop1'].prop2)
+     * @returns {*|undefined} The value of the objects property or undefined if the property doesn't exist
+     */
+    function getProp (obj, prop) {
```

Comment:
Please use the lodash _.get function instead:

![image](https://user-images.githubusercontent.com/6422711/94803565-c60df980-03e9-11eb-8f7a-e1ce68eb52fb.png)